source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -1961,6 +1961,9 @@ AdaptiveNamespaceStart
         [propget] HRESULT SupportsInteractivity([out, retval] boolean* value);
         [propput] HRESULT SupportsInteractivity([in] boolean value);
 
+        [propget] HRESULT ImageBaseUrl([out, retval] HSTRING* value);
```

Comment:
>ImageBaseUrl [](start = 26, length = 12)

Shouldn't we be consistent in our usage of URI and URL?

Reply:
I guess URI would be a better (more general) term. Do you think we should change in other places as well? For example, all source names in json payload use URL (`Image`'s url, `Action`'s iconUrl) and shared objects' variables and functions use url as well [Image.h](https://github.com/Microsoft/AdaptiveCards/blob/master/source/shared/cpp/ObjectModel/Image.h#L18)

Reply:
renamed in 2d6f60b